### DISCUSSION TITLE
verify error 17 times

### DISCUSSION THREAD

**Discussion by Adeshen** [community/community#52300] [GitHub Education]
I have upload all the need education material change my profile to my real name
But it always says that
"Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Apr 8, 2023 for [@Adeshen](https://github.com/Adeshen).

You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio."

**Comment by jamesopppp** [comment]
me too

**Comment by samus-aran** [comment]
Hi @Adeshen 👋 

Here is a quick tip on submitting evidence:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.
- We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

Hope this helps! Let us know how you get along 👍
